tests/test_peak_detection.py

```diff
@@ -1193,3 +1195,12 @@ def test__A1_data_causing_TwoValleysInARowError(MA20123123__2020_10_13_234733__A
     # fmt: on
     assert np.array_equal(peak_indices, expected_peak_indices)
     assert np.array_equal(valley_indices, expected_valley_indices)
+
+
+def test__D4_data_causing_TwoValleysInARowError(MA202000030__2020_12_11_233215__D4):
+    _, peak_and_valley_indices = MA202000030__2020_12_11_233215__D4
+    peak_indices, valley_indices = peak_and_valley_indices
+    expected_number_peaks = 15
+    expected_number_valleys = 15
```

Comment:
so I know anna you specifically asked me about what to assert and I just said the number of peaks.  

but reflecting more though, I think once it's established which peak is the correct one, (i.e. which of the two in a row in brandon's we think is the best one to be chosen, or which one was missing from the customer data file and is now correctly present), we can add an assertion here as a "snapshot test" after we know the correct value so that we would be alerted in the future if it accidentally changed